examples/SensorDemo/SensorDemo.ino

```diff
@@ -0,0 +1,96 @@
+#include <SPI.h>
+#include <SPBTLE_RF.h>
+#include <sensor_service.h>
+
+/* Configure SPI3:
+  MOSI: PC12
```

Comment:
Specific to disco IOT ? rename example or add comment on top of the scketch to specify it

Reply:
same comment as on the other example.

In addition, on this one, I can see the temperature but no success yet to have the accelero working.
And the units for humidity and pressure is not good.

Reply:
The application doesn't work (wrong units display). So we have change for BlueNRG from ST. 
And on the old apps, don't forget to enable the notification for the accelero.